### ISSUE TITLE
Model resulted in Nan value during calibration.

### ISSUE THREAD

**Issue by tu1258** [PINTO0309/PINTO_model_zoo#43]
Code (just a demo how I do quantize and it can't reproduce error)

```
def representative_dataset_gen():
    for x in validation_fingerprints:
      x = x[np.newaxis,:]
      yield [x]

converter = tf.lite.TFLiteConverter.from_saved_model(flags.train_dir + '/last_model')
converter.optimizations = [tf.lite.Optimize.DEFAULT]
converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS_INT8]
# converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS]
converter.allow_custom_ops = True
converter.inference_input_type = tf.uint8
converter.inference_output_type = tf.uint8
converter.representative_dataset = representative_dataset_gen
last_quant_model = converter.convert()
with open(flags.train_dir + '/quant_last_model.tflite', 'wb') as w:
  w.write(last_quant_model)

```
Some config
type(validation_fingerprints): <class 'numpy.ndarray'>
shape(validation_fingerprints): (3093, 16384)
type(x): <class 'numpy.ndarray'>
shape(x): (1,16384)
The model_summary
[model_summary.txt](https://github.com/PINTO0309/PINTO_model_zoo/files/5405198/model_summary.txt)

`Validation_fingerprints` is np.float32. I don't know if it would cause problem in full integer quantization. (I found that [4-2-6-7. Full Integer Quantization from saved_model (All 8-bit integer quantization)](https://qiita.com/PINTO/items/865250ee23a15339d556#4-2-6-7-full-integer-quantization-from-saved_model-all-8-bit-integer-quantization) also use np.float32 tho.)
I've also found [this issue](https://github.com/tensorflow/tensorflow/issues/22803) but setting `fused=False` in batch norm doesn't help. Is there any advice? Stuck this for several days 😢

**Comment by noahzhy** [comment]
did you fixed it?
